frontend/src/components/GenreList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Genre from "./Genre";
+
+const GenresList = [
```

Comment:
Please separate your data from your component, as seen on Monday with Laurent
The idea is to be able to reuse your data file across your whole project

Reply:
data is separate ;)